test/test_khard.py

```diff
@@ -44,3 +46,39 @@ def test_no_search_terms_result_in_any_queries(self):
                              source_search_terms=query.AnyQuery(),
                              target_contact=query.AnyQuery())
         self.assertEqual(expected, prepared["foo"])
+
```

Comment:
PEP-8 line length please.

Reply:
Yep, is there a standard auto-formatter you'd recommend for this project?

Reply:
No, I have a language server for python in my editor and have a marker for overlong lines.